src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {

```

Comment:
The class IsoscelesTrapezoid should not start with an empty line. Please remove the empty line to follow the best practices.